src/mailing/sender.js

```diff
@@ -43,5 +45,34 @@ async function sendTestEmail(to, username) {
   });
 }
 
-// Export the transport and the sendTestEmail function
-module.exports = { sendTestEmail };
+async function sendForgotPasswordEmail(toEmail, resetToken) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.EMAIL_USER,
```

Comment:
The value in the `.env` file that I see in the team slack is `EMAIL_USERNAME`: https://codethedream.slack.com/archives/C05G08542H3/p1693435063248899

Reply:
Sorry, you are right 👍